.azure-pipelines/daily-build.yml

```
@@ -9,20 +9,148 @@ schedules:
 
 pr: none
 
-pool:
-  vmImage: 'windows-2019'
-
-steps:
-- task: DotNetCoreCLI@2
-  displayName: Build for Version Bump
-  inputs:
-    command: 'custom'
-    custom: 'msbuild'
-    arguments: 'build.proj'
-
-- task: ComponentGovernanceComponentDetection@0
-  displayName: Component Detection
-  inputs:
-    scanType: 'Register'
-    verbosity: 'Normal'
-    alertWarningLevel: 'High'
\ No newline at end of file
+jobs:
+- job: DailyBuild
+  timeoutInMinutes: 180
+  pool:
+    vmImage: 'windows-2019'
+  steps:
+  - task: PowerShell@2
+    displayName: 'Initialization'
+    inputs:
+      targetType: 'inline'
+      script: |
+        Set-PSRepository -Name PSGallery -InstallationPolicy Trusted
+        # Avoid issue of filename too long
+        git config --global core.longpaths true
+        # Wipe any changes from a previous branch
+        git clean -xdf
+
+  - task: PowerShell@2
+    displayName: Prepare Powershell $(PSVersion) 
+    inputs:
+      filePath: 'tools/Test/SmokeTest/PrepareRequiredPowershell.ps1'
+      arguments: '-RequiredPsVersion $(PSVersion)'      
+
+  - task: PowerShell@2
+    displayName: 'Install platyPS'
+    inputs:
+      targetType: 'inline'
+      script: |
+        $command = "`$PSVersionTable `
+                    Install-Module platyPS -Force -Confirm:`$false -Scope CurrentUser `
+                    Exit"
+        dotnet tool run pwsh -c $command
+
+  - task: DotNetCoreCLI@2
+    displayName: 'Build for Bumping Version'
+    inputs:
+      command: custom
+      custom: msbuild
+      arguments: 'build.proj /t:"Build" /p:"Configuration=Release"'
+
+  - task: PowerShell@2
+    displayName: 'Bump Version'
+    inputs:
+      targetType: 'inline'
+      script: |
+        $command = "`$PSVersionTable `
+        ./tools/RunVersionController.ps1  -MonthName $(MonthName) -Year $(Year) `
+        Exit"
+        dotnet tool run pwsh -c $command  
+
+  - task: PowerShell@2
+    displayName: 'Push to Branch internal/dailybuild'
+    inputs:
+      targetType: 'inline'
+      script: |
+        git config user.email "bez@microsoft.com"
+        git config user.name "azuresdkci"
+        git remote set-url origin https://$(Token)@github.com/Azure/azure-powershell.git
```

Try to use bot's token
```suggestion
        git remote set-url origin https://$(Token)@github.com/Azure/azure-powershell.git
```